src/bokeh/models/transforms.py

```diff
@@ -108,13 +108,13 @@ def __init__(self, *args, **kwargs) -> None:
     present in the function namespace at render time. The snippet will be
     into the body of a function and therefore requires a return statement.
 
-    Example:
+    Example
 
-        .. code-block:: javascript
+    .. code-block:: javascript
 
-            func = '''
-            return Math.floor(x) + 0.5
-            '''
+        func = '''
+        return Math.floor(x) + 0.5
+        '''
```

Comment:
- [Current docs](https://docs.bokeh.org/en/latest/docs/reference/models/transforms.html#bokeh.models.CustomJSTransform.func)

![grafik](https://github.com/bokeh/bokeh/assets/68053396/266e7978-6809-43a5-8b34-76ffd011f13e)

- local build

![grafik](https://github.com/bokeh/bokeh/assets/68053396/b9f88d6e-e7b4-4322-b1cc-e10686e2e029)

Reply:
Where is the purple highlight coming from? is that just a search in your browser or actually part of the docs?

Reply:
This is only the search engine of my browser and has nothing to do with the documentation.